src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -209,11 +209,11 @@ namespace JSONHelpers
         return true;
     }
 
-    void FancyZonesData::SetActiveZoneSet(const std::wstring& deviceId, const std::wstring& uuid)
+    void FancyZonesData::SetActiveZoneSet(const std::wstring& deviceId, const ZoneSetData& data)
     {
-        if (!uuid.empty() && deviceInfoMap.find(deviceId) != deviceInfoMap.end())
+        if (deviceInfoMap.find(deviceId) != deviceInfoMap.end())
```

Comment:
Maybe we can re-organize it like:
```
auto it = deviceInfoMap.find(deviceId);
if (it != deviceInfoMap.end()) {
    it->second.activeZoneSet = data;
}
```
To avoid searching for `deviceId` inside map twice.